Problem1.java

```diff
@@ -0,0 +1,48 @@
+// Time complexity: O(n)
```

Comment:
LC says the time complexity for this is O(n^2). You are copying lists in the end and the time to compy that is O(n). Worst case it will be O(n^2).